releases/release-1.22/release-notes/maps/pr-102297-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 102297
+releasenote:
+  text: '`--ssh-user` and `--ssh-key` options are removed. They only functioned on
```

Comment:
This looks like it can cause problems in future runs of krel release-notes like the one we hit in: https://github.com/kubernetes/sig-release/pull/1593

Reply:
This is a valid yaml file(I double checked in a yaml file validator). I could rephrase for sure ^^